tests/builtins/test_bytes.py

```diff
@@ -7,24 +7,21 @@ class BytesTests(TranspileTestCase):
 
 class BuiltinBytesFunctionTests(BuiltinFunctionTestCase, TranspileTestCase):
     functions = ["bytes"]
+    small_ints = True
+
+    def test_set(self):
+        self.assertCodeExecution("""
+            x = {66}
+            print(bytes(x))
+            """)
+
+    def test_frozenset(self):
+        self.assertCodeExecution("""
+            x = frozenset((66))
```

Comment:
Are you sure this example is testing what you think it is? Doesn't it need a comma after the 66 to make this a "set of a tuple"? Without the comma, it should raise an error -- which it may well do on both platforms, making the test pass, but that's not what I think you *think* you're testing.